source/languages/en/riakcs/cookbooks/configuration/Configuring-Stanchion.md

```diff
@@ -8,60 +8,94 @@ audience: intermediate
-appear in the `stanchion` config section of the file, which looks like
-this:
+You can set the IP using the `listener` parameter. Replace `127.0.0.1` with the
+IP address of the Stanchion node, and `8080` with the port of the Stanchion
+node.
+
+```stanchionconf
+listener = 127.0.0.1:8080
+```
 
 ```appconfig
 {stanchion, [
-
-    %% Configs here
-
-]}
+             {stanchion_ip, "127.0.0.1"},
+             {stanchion_port, 8080}
+             %% Other configs
+            ]}
 ```
 
-
-You can set the IP using the `stanchion_ip` parameter. Replace
-`127.0.0.1` with the IP address of the Stanchion node.
-
-<div class="note">
-<div class="title">Note on matching IP addresses</div>
-The IP address you enter here must match the IP address specified for
-the <code>stanchion_ip</code> variable in the Riak
-<code>app.config</code> file and the Riak CS <code>app.config</code>
-file.
+<div class="note"><div class="title">Note on matching IP addresses</div>
+The IP address you enter here must match the IP address specified for the
+<code>stanchion_host</code> variable in the Riak <code>riak.conf</code> file and
+the Riak CS <code>riak-cs.conf</code> file.
 </div>
 
-If you want to use a different port for Stanchion to accept connections
-on, you must change the `stanchion_port` setting. Replace `8085` with
-the port number you want to use.
+If you want to use SSL, make sure the `ssl.certfile` and `ssl.keyfile` settings
+are not commented out, and have been set correctly.
 
-The `stanchion_ssl` variable is set to `false` by default. If you want
-to use SSL, change this variable to `true`.
+```stanchionconf
+ssl.certfile = "./etc/cert.pem"
+ssl.keyfile = "./etc/key.pem"
+```
+
+```appconfig
+{stanchion, [
+             {ssl, [
+                    {certfile, "./etc/cert.pem"},
+                    {keyfile, "./etc/key.pem"}
+                   ]},
+             %% Other configs
+            ]}
+```
 
 ## Specifying Riak Information
 
-If you are running a single node for experimentation, or if a Riak node
-is running locally and configured to listen for protocol buffer traffic
-on `0.0.0.0`, the default Riak configuration for Stanchion should be
-fine.
+If you are running a single node for experimentation, or if a Riak node is
+running locally and configured to listen for protocol buffer traffic on
+`0.0.0.0`, the default Riak configuration for Stanchion should be fine.
 
-Otherwise, update the IP address and port for Riak in the Stanchion
-`app.config` file. The settings appear in the `stanchion` section:
+Otherwise, update the IP address and port for the Riak host in the Stanchion
+configuration file.
 
-* `riak_ip` --- Replace `127.0.0.1` with the IP address of the Riak node
-* `riak_pb_port` --- Replace `8087` with the port number set in the Riak
-  `app.config` file
+```stanchionconf
+riak_host = 127.0.0.1:8087
+```
+
+```appconfig
+{stanchion, [
+             {riak_ip, "127.0.0.1"},
+             {riak_pb_port, 8087}
```

Comment:
ditto ... These are changed to `{riak_host, {"127.0.0.1", 8087}}" even in `app.config` .